src/borg/crypto/key.py

```diff
@@ -442,6 +445,18 @@ def env_passcommand(cls, default=None):
                 raise PasscommandFailure(e)
             return cls(passphrase.rstrip('\n'))
 
+    @classmethod
+    def fd_passphrase(cls):
+        try:
+            fd = int(os.environ.get('BORG_PASSPHRASE_FD', None))
+        except (ValueError, TypeError):
+            return None
+        with os.fdopen(fd) as f:
```

Comment:
it is the default, but maybe it would be clearer here to give mode='r' - just to point out we're using text mode (resulting in `str`) and not binary mode (resulting in `bytes`).

Reply:
Done.